src/pages/workspace/taxes/WorkspaceCreateTaxPage.tsx

```diff
@@ -117,6 +119,10 @@ function WorkspaceCreateTaxPage({
                                     // We increase the amount max length to support the extra decimals.
                                     amountMaxLength={CONST.MAX_TAX_RATE_INTEGER_PLACES}
                                     extraSymbol={<Text style={styles.iouAmountText}>%</Text>}
+                                    autoGrowExtraSpace={variables.w80}
+                                    autoGrowMarginSide="left"
+                                    // Currently, autoGrow is only supported on Android. Therefore, we need to check the platform before applying flex styles to avoid layout issues on other platforms.
+                                    style={[styles.iouAmountTextInput, styles.textAlignRight, getPlatform() === CONST.PLATFORM.ANDROID && styles.flex1]}
```

Comment:
I'm curious about the `styles.flex1` here, why don't we just use `autoGrowExtraSpace={variables.w80}`?

Reply:
https://github.com/Expensify/App/pull/59078 caused https://github.com/Expensify/App/issues/60765 and there's a fix here.
The author prepared the fix: https://github.com/Expensify/App/pull/60842
Can you look at the [PR](https://github.com/Expensify/App/pull/60842) and let me know if we still need `styles.flex1` for Android? TIA

Reply:
Yeah. We should do that 😄

Reply:
Merged now 😬

Reply:
@linhvovan29546 Can you merge main then verify this change? Ty